core/client/verify.go

```diff
@@ -55,8 +55,8 @@ func NewVerifier(vrf vrf.PublicKey,
 	}
 }
 
-// NewVerifierFromConfig creates a new instance of the client verifier from a config.
-func NewVerifierFromConfig(config *pb.Domain) (*Verifier, error) {
+// NewVerifierFromDomain creates a new instance of the client verifier from a config.
```

Comment:
nit: 'from a domain config' ?  (i.e. `config` contains data used to configure a `Domain`) wdyt?

Reply:
Done